### DISCUSSION TITLE
logging proposal

### DISCUSSION THREAD

**Discussion by JustinGuese** [caprover/caprover#2040] [Ideas]
Hey guys, loving caprover so far!
What do you think about an integration of "logging" into the app similar to the monitoring?
Currently I am running "openobserve", which is a lightweight Elasticsearch + kibana, and using Filebeat with simply reading out the docker socket for container logs... 
I think the setup itself is relatively simple, with a great benefit of having logs, errors, warnings shown in the openobserve logs?
I could probably implement the backend functionality, but would need help with the frontend

**Comment by githubsaturn** [comment]
Hi Justin! Thanks for your kind words!

openobserve could be one of the one click apps that you install on CapRover. Why would you think this needs to be integrated in the core?
